cephci/cluster_info.py

```diff
@@ -152,12 +160,43 @@ def gather_info(cluster):
     return info
 
 
+def get_ceph_logs(cluster, log_dir):
+    """Capture ceph cluster logs"""
+    for node in cluster.get_nodes():
+        download_dir = os.path.join(log_dir, "ceph_logs", node.hostname)
+        os.makedirs(download_dir, exist_ok=True)
+        files = node.get_dir_list(dir_path=CEPH_VAR_LOG_DIR, sudo=True)
+        attributes = node.get_listdir_attr(dir_path=CEPH_VAR_LOG_DIR, sudo=True)
+        log.info(f"List ceph var log files : {files}")
+        for file, attribute in zip(files, attributes):
+            if attribute.filename == file:
+                if S_ISDIR(attribute.st_mode):
+                    os.makedirs(os.path.join(download_dir, file), exist_ok=True)
+                    ceph_files = node.get_dir_list(
+                        dir_path=os.path.join(CEPH_VAR_LOG_DIR, file), sudo=True
+                    )
+                    for ceph_file in ceph_files:
+                        node.download_file(
```

Comment:
Instead of calling download method twice, I would sugest to collect filenames first and download all files

Reply:
files are getting downloaded 1 by 1